update-gemfile.sh

```diff
@@ -3,7 +3,7 @@
 COMPOSE=podman-compose
 if ! which "$COMPOSE" 2>&1 1>/dev/null
 then
-	COMPOSE=docker-compose
```

Comment:
This... is actually potentially dangerous. But ok if you tested it (I can't, I use podman).

Reply:
yeah ... with `docker-compose` there seems to be an error now: https://github.com/quarkusio/quarkusio.github.io/pull/2056#issue-2435011818